pants.toml

```diff
@@ -12,6 +12,21 @@ backend_packages.add = [
   "pants.backend.python.typecheck.mypy",
 ]
 
+# Pants' sponsor, Toolchain, offers remote caching, which can improve your CI performance
+# with minimal configuration. For more details, visit https://toolchain.com
+
+plugins.add = [
+  "toolchain.pants.plugin==0.20.0",
```

Comment:
Thoughts on this being commented out? This is going to result in people installing things they don't need, and subsystems showing up in `./pants help`.

I'm worried about past issues we've had where the Toolchain plugin requirements caused Pants to not install. cc @stuhood 

Ack that you would need to then comment out the sections on `toolchain-setup` and `auth`. I think that's worth it.